model/generate.go

```diff
@@ -278,6 +282,19 @@ func (g *GeneratedProject) saveNewBuildsAndTasks(ctx context.Context, cachedProj
 	return nil
 }
 
+func (g *GeneratedProject) findVariantsToActivate() []string {
+	var toActivate []string
+	for _, bv := range g.BuildVariants {
+		if bv.BatchTime == nil {
```

Comment:
I think this should also check bv.CronBatchTime

Reply:
fixed